packages/scene-composer/src/components/panels/scene-components/DataOverlayComponentEditor.spec.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { act, render } from '@testing-library/react';
+import wrapper from '@awsui/components-react/test-utils/dom';
+
+import { KnownComponentType } from '../../../interfaces';
+import { IDataOverlayComponentInternal, ISceneNodeInternal, useStore } from '../../../store';
+import { Component } from '../../../models/SceneModels';
+import { mockProvider } from '../../../../tests/components/panels/scene-components/MockComponents';
+
+import { DataOverlayComponentEditor } from './DataOverlayComponentEditor';
+
+jest.mock('@awsui/components-react', () => ({
+  ...jest.requireActual('@awsui/components-react'),
+}));
+
+jest.mock('./data-overlay/DataBindingMapEditor', () => {
+  const originalModule = jest.requireActual('./data-overlay/DataBindingMapEditor');
+  return {
+    ...originalModule,
+    DataBindingMapEditor: (...props: unknown[]) => {
```

Comment:
Why the type is **unknown** here?

Reply:
the type of data in it doesn't matter for this test file, so I just put unknown for now